storage/battlesystem/battlesystem.yaml

```diff
@@ -0,0 +1,15 @@
+hero:
+  HP: 60
+  ATK: 40
+  DEF: 40
+  SPA: 40
+  SPD: 40
+  SPE: 40
+  Moves:
+  - Name: slash
+    MovesType: atk
+    Power: 60
+    Accuracy: 80
+    Effects:
+    - atkall
+    - defdown
```

Comment:
instead of battlesystem/battlesystem.yaml that contains all the fighters ever, it could save the fighters separately as their own unique entries, like hero.yaml. Formatting can be the same.

This is to match how the character data is saved

Reply:
Done!